modules/snooker/src/test/ts/browser/CopyColumnsTest.ts

```diff
@@ -17,7 +17,7 @@ UnitTest.test('CopyColumnsTest', () => {
     Insert.append(SugarBody.body(), table);
 
     const rowsOpt = copyCols(table, {
-      selection: [ Hierarchy.follow(table, [ section, row, column, 0 ]).getOrDie(label + ': could not follow path') ]
+      selection: [Hierarchy.follow(table, [section, row, column, 0]).getOrDie(label + ': could not follow path')]
```

Comment:
Something seems to have gone wrong here, as this should have the extra spaces.
```suggestion
      selection: [ Hierarchy.follow(table, [ section, row, column, 0 ]).getOrDie(label + ': could not follow path') ]
```